test/e2e-tests.sh

```diff
@@ -82,6 +82,43 @@ function test_setup() {
   $(dirname $0)/upload-test-images.sh e2e || fail_test "Error uploading test images"
 }
 
+readonly PUBSUB_SERVICE_ACCOUNT="eventing_pubsub_test"
```

Comment:
Please move these 3 vars to the top. Also add a comment about what they are.

Reply:
Done.